### PR TITLE
Python 3 Support

### PR DESCRIPTION
**Pull request by flavioab**
- Fixed unit tests
- Fixed wrong Rastreio webservice url
- Applied changes from using 2to3 python conversion

### CODE REVIEW

**Comment by flavioab** [comment]
@idontdomath @aleciten

File: pysigep/rastreamento/consulta_rastreamento.py [hunk 1]
```diff
@@ -75,9 +75,6 @@ def __init__(self):
 
     def _parse_xml(self, xml):
 
-        # Necessario pois o decode do rastreamento é diferente
-        xml = xml.decode('utf8').encode('iso-8859-1')
```
Comment (gseva) [inline pysigep/rastreamento/consulta_rastreamento.py:79]: Care as this _may_ be still needed in python 3

Comment (gseva) [inline pysigep/rastreamento/consulta_rastreamento.py:79]: This should do the same in python 3:

``` python
xml = bytes(xml,'utf-8').decode('iso-8859-1')
```

**Comment by aleciten** [comment]
Why do we have that coveralls user?!

**Comment by aleciten** [comment]
Ahh, public repo because of the fork

**Comment by gseva** [comment]
🐑

**Pull request merged**